README.md-review

```diff
@@ -0,0 +1,107 @@
+# Operator for IBM block storage CSI driver
+The Container Storage Interface (CSI) Driver for IBM block storage systems enables container orchestrators such as Kubernetes to manage the life-cycle of persistent storage.
+
+This is the official operator to deploy and manage IBM block storage CSI driver.
+
+Supported container platforms:
+  - Openshift v4.1
+  - Kubernetes v1.13
+
+Supported operating systems:
+  - RHEL 7.x (x86 architecture)
+  - RHCOS 4.1 (x86 architecture)
+
+## Prerequisite
+Please go to section [`Prerequisites for Driver Installation`](https://github.com/IBM/ibm-block-csi-driver#prerequisites-for-driver-installation) for details.
+
+## Installation
+
+### Install the operator
+1. Download the manifest from GitHub.
+```bash
+curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/develop/deploy/ibm-block-csi-operator.yaml > ibm-block-csi-operator.yaml
+```
+2. Update the image fields in the ibm-block-csi-operator.yaml if the default values are not fit for you.
```

Comment:
(Optional): If required, update the image fields in the ibm-block-csi-operator.yaml.